views/templates/panes/compiler.pug

```diff
@@ -22,6 +22,21 @@ mixin newPaneButton(classId, text, title, icon)
               .argmenuitem
                 span.argtitle Detailed Compiler Flags
                 span.argdescription Open a new window to edit verbose compiler flags
+    .btn-group.btn-group-sm.w-100(role="group" aria-label="Compiler revision picker")
+      noscript This feature requires JavaScript to be enabled.
+      .input-group.compiler-revision-picker
+        <!-- Selector to be wrapped in TomSelect -->
+        select
+        .input-group-append
+          <!--- Naviate backward in revision history -->
```

Comment:
also copy and pasted typo in these comments for `Navigate`

Reply:
Done, thanks